spec/classes/python_spec.rb

```diff
@@ -217,6 +217,22 @@
     end
   end
 
+  context "on a Redhat 7 OS" do
```

Comment:
awesome work, thank you!

Would you mind adding a case for RedHat 6, that shows the same thing as this, but with the correct pip package name and update the existing RedHat 5 part with the name attribute. That would give us much better testing for all use cases with EL.